### PR TITLE
211 sponsors footer

### PR DESCRIPTION
**Pull request by julianahhong**
create sponsors section and footer

### CODE REVIEW

File: public/css/index.less [hunk 1]
```diff
@@ -577,14 +542,18 @@ section {
...
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+  margin-top: 30px;
 }
 
 footer p {
   color: #f8f8f8;
+  text-align: center;
+
```
Comment (mahsu) [inline public/css/index.less]: remove

File: public/css/index.less [hunk 2]
```diff
@@ -40,7 +40,7 @@ html {
 body {
   width: 100%;
   height: 100%;
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Avenir', sans-serif;
```
Comment (mahsu) [inline public/css/index.less]: are these fonts added in a different pr?

File: public/css/index.less (hunk 1 above)
Comment (julianahhong) [inline public/css/index.less]: are you just referring to the extra line here or the "text-align: center"?

Comment (mahsu) [inline public/css/index.less]: extra line, sorry.

File: public/css/index.less (hunk 2 above)
Comment (julianahhong) [inline public/css/index.less]: let me add/change the font in the main.less file

**Pull request merged**